### PR TITLE
[WIP] More dynamic things

### PR DESCRIPTION
**Pull request by KrauseFx**
:100:

### CODE REVIEW

**Comment by KrauseFx** [comment]
:white_check_mark: | No errors found
  ------------- | ------------ 



  :white_check_mark: | No warnings found
  ------------- | ------------ 



  &nbsp; | 2 Messages
  ------------- | ------------ 
    :book: | Trusted author @KrauseFx
    :book: | Tests are updated / added


<p align="right">
  Generated by 
    <a href="https://github.com/KrauseFx/danger">danger</a>
  on 
  <i>2016-01-04</i>
</p>

File: lib/danger/available_values.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      ]
+    end
+
+    def self.request_source
+      [
+        :pr_title,
+        :pr_body,
+        :pr_author
+      ]
+    end
```
Comment (orta) [inline lib/danger/available_values.rb]: I like these ideas! Ideally there should be a way to apply rubydoc so we can generate documentation from this file for the README too

Comment (KrauseFx) [inline lib/danger/available_values.rb]: Agreed :+1:

**Comment by KrauseFx** [comment]
:rocket:

**Pull request merged**